src/ui/components/Button/Button.scss

```diff
@@ -1,53 +1,28 @@
-@import '~core/css/inc/mixins';
-@import '~ui/css/vars';
+@import "~core/css/inc/mixins";
+@import "~ui/css/vars";
 
 .Button--neutral {
-  @include button(
-    $background: $neutral-base-color,
-    $background-active: $neutral-active-color,
-    $background-hover: $neutral-hover-color
-  );
+  @include button($background: $neutral-base-color, $background-active: $neutral-active-color, $background-hover: $neutral-hover-color);
```

Comment:
Do all of these need to go on one line? I found it easier to read when they were split across four lines. It's also more friendly to split screen editors.